tests/conftest.py

```diff
@@ -418,12 +422,30 @@ def fake_dists(tmpdir, make_package, make_wheel):
         make_package("small-fake-a", version="0.1"),
         make_package("small-fake-b", version="0.2"),
         make_package("small-fake-c", version="0.3"),
+        make_package("small-fake-d", version="0.4"),
+        make_package("small-fake-e", version="0.5"),
+        make_package("small-fake-f", version="0.6"),
```

Comment:
Recently, I've decreased this list down to 3 packages for the sake of the tests speed, see #1735. Would it be possible to keep this list as small as possible? What do you think?

Reply:
Yes. I have also made sure that the extra packages required for the build requirement tests are not available to existing tests.